docs/core/operators.rst

```diff
@@ -376,6 +382,8 @@ We don't do it this way because "the set of all possible clock values" is over 8
 
 CHOOSE is useful whenever we need to pull a value from a set.
 
+.. todo:: Can I do things like that in real, big models? That question immediately jumps to my mind, is it practical or just cute (like the naive Haskell quicksort everybody loves)?
```

Comment:
Yeah, that's the idiomatic way to do it. CHOOSE is almost never the bottleneck, either it's a really hairy computation or an absolutely massive state space. You're almost never choosing over an 80,000 element set, either, usually it's 10-20